spec/lib/pact_broker/domain/webhook_request_spec.rb

```diff
@@ -49,6 +49,45 @@ module Domain
         end
       end
 
+      describe "redacted tokens" do
```

Comment:
And again!

Reply:
👍

Reply:
tests moved into `spec/lib/pact_broker/webhooks/url_redactor_spec.rb`